panda/src/glstuff/glBufferContext_src.cxx

```diff
@@ -47,3 +47,10 @@ evict_lru() {
   update_data_size_bytes(0);
   set_resident(false);
 }
+
+/**
+ * Returns the native OpenGL buffer ID for this buffer.
+ */
+uint64_t CLP(BufferContext)::get_native_id() const {
```

Comment:
Nitpick: Panda style is to put the part starting with `get_native_id` on a new line. See the other definitions in this file.